features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/data/AuthManager.jsx

```diff
@@ -156,6 +156,18 @@ class AuthManager {
             });
     }
 
+    /**
+     * retrieve Settings from settings rest api and store in the local storage
+     */
+    static setSettings() {
+        const promisedResponse = fetch('/api/am/store/v1.0/settings', {});
```

Comment:
Why we store it in localstorage ???

Reply:
I have inserted applicationGroupSharingEnabled field to the settings api and then i store that information on local storage since i need to refer that info when rendering /application related UIs. Also, for each page refresh i don't have to do a network call to retrieve that info since i am storing in local storage.

Reply:
resolved in the latest code